examples/basics/visuals/text_visual.py

```diff
@@ -1,23 +1,77 @@
-# -*- coding: utf-8 -*-
-# vispy: gallery 30
-# -----------------------------------------------------------------------------
-# Copyright (c) Vispy Development Team. All Rights Reserved.
-# Distributed under the (new) BSD License. See LICENSE.txt for more info.
-# -----------------------------------------------------------------------------
-
```

Comment:
Why remove this comment block?

Reply:
Obs, sry that wasn't intentional. I just headlessly overwrote the original example. I'll put it back in of course.